tests/integrations/logging/test_logging.py

```diff
@@ -136,6 +137,30 @@ def filter(self, record):
     assert event["logentry"]["message"] == "hi"
 
 
+def test_logging_captured_warnings(sentry_init, capture_events):
+    sentry_init(
+        integrations=[LoggingIntegration(event_level="WARNING")],
+        default_integrations=False,
+    )
+    events = capture_events()
+
+    logging.captureWarnings(True)
+    warnings.warn("first")
+    warnings.warn("second")
+    logging.captureWarnings(False)
+
```

Comment:
Please add another `warning.warn()` and test below that this `warnings.warn()` was not captured in the events. Thanks

Reply:
Done ✅

Reply:
Nice.